modules/runtime/pages/purge-tool.adoc

```diff
@@ -2,65 +2,207 @@
+----
+./bin/bonita-purge-tool delete --older-than 6m --process-definition-id 1234567890
+----
 
+Windows::
++
 [source,shell]
 ----
-# If no PROCESS_DEFINITION_ID parameter is given, the program lists all existing process definitions and stops:
-bin/bonita-purge-tool.bat
+./bin/bonita-purge-tool.bat delete --older-than 6m --process-definition-id 1234567890
+----
+====
 
-bin/bonita-purge-tool.bat <PROCESS_DEFINITION_ID> <OLDEST_DATE_TIMESTAMP> [<TENANT_ID>]`
+.Delete archived process instances before 5 July 2022
+[tabs]
+====
+Unix::
++
+[source,shell]
+----
+./bin/bonita-purge-tool delete --before-date 1656979200000
 ----
 
-An optional *TENANT_ID* parameter can be given if the platform uses multiple tenants to specify on which tenant should the process instances be deleted.
-If multi-tenancy is used and if the TENANT_ID is not set, an error is issued, and the program stops.
+Windows::
++
+[source,shell]
+----
+./bin/bonita-purge-tool.bat delete --before-date 1656979200000
+----
+====
+NOTE: The `--before-date` parameter must be in https://www.epochconverter.com/[milliseconds since the epoch].
+
+==== Delete modes
+
+===== `batch-delete`
+
+By default, the tool uses the `batch-delete` mode to delete rows in database tables. +
+This mode will be slower than the copy-truncate mode, but it doesn't require the Runtime to be shutdown and can be stopped and resumed at any time.
+
+In this mode, each deleted batch is committed in database. +
+The batch size and the timeout interval between each batch can be configured using the `--batch-size` and `--timeout-interval` options on the `delete` command.
+
+Fine tune the batch size and timeout interval depending on your database configuration and the volume of data to delete. By default, the batch size is `5000` and the timeout interval is `0` ms.
+
+===== `copy-truncate`
+
+In this mode, the tool will copy the rows to keep in a temporary table, then truncate and drop the original table and rename the temporary table and recreate all required constraints and indices.
+
+Due to the efficiency of the TRUNCATE command, this method should be faster in most cases, especially when the number of rows to delete is higher than the row to keep in the table, but it requires the Runtime to be stopped during the operation.
+
+Use the `--delete-mode copy-truncate` option to use this mode with the `delete` command.
 
-*OLDEST_DATE_TIMESTAMP* must be a Timestamp (in milliseconds) from which all process instances that finished before that date will be deleted. +
-Unfinished process instances and process instances that finished after that date will not be affected. +
-Its format is a standard Java timestamp since https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/time/Instant.html#EPOCH[EPOCH] (in milliseconds).
-You can use websites such as https://www.epochconverter.com/[Epoch Converter] to format such a timestamp.
+=== `delete-file-input` command
+
+Delete all archived contract file input values.
```

Comment:
```suggestion
Delete all archived contract file input values. +
In other words, delete all rows in table `arch_contract_data` corresponding to contract data of type `File` (in Studio) or `org.bonitasoft.engine.bpm.contract.FileInputValue` in Bonita Engine. +
These data are not used by Bonita and can take a large amout of space in your database, so deleting them is advised.

[NOTE]
====
From Bonita 10.4, these data are no more archived, so it will not be necessary anymore to run this command.
====

```